project/AppDependencies.scala

```diff
@@ -3,7 +3,7 @@ import sbt._
 
 object AppDependencies {
 
-  val bootStrapPlayVersion = "2.0.0"
+  val bootStrapPlayVersion = "2.2.0"
```

Comment:
I think it should be bootstrap-frontend-play-26 instead of bootstrap-play-26 if we are doing the bootstrap upgrade for this and this should be bumped to 3.2.0. This can be found [here](https://github.com/hmrc/bootstrap-play#configuration). Also can you please bump scalatest to 3.2.3. Thanks

Reply:
why should it be changed to bootstrap fronted?

Reply:
Sorry for not being clear. It says in the story we are moving from bootstrap-play-26 to bootstrap-play [here](https://jira.tools.tax.service.gov.uk/browse/MTDSA-7743) so if you check the README for the bootstrap-play I sent earlier, it is bootstrap-frontend-play-26 as this is a front end service for bootstrap-play and the latest version is 3.2.0. For a backend like our apis we use bootstrap-backend-play-26 which can be seen on [here](project/AppDependencies.scala) on one of the PRs open for bootstrap upgrades